charts/kured/templates/clusterrole.yaml

```diff
@@ -9,9 +9,9 @@ rules:
 # Allow kured to read spec.unschedulable
 # Allow kubectl to drain/uncordon
 #
-# NB: These permissions are tightly coupled to the bundled version of kubectl; the ones below
-# match https://github.com/kubernetes/kubernetes/blob/v1.19.4/staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go
```

Comment:
Is the link useful for figuring out which apigroups, etc? If yes, we should probably keep it around?

Reply:
Isn't the other text good enough? I am reluctant to keep it to a single version, but I think I can live with it.

Reply:
Ok let me rollback that change, and keep it as is.

Reply:
Done.